python-sdk/src/astro/extractors/facets.py

```diff
@@ -0,0 +1,55 @@
+from typing import Dict, List, Optional
+
+import attr
+from openlineage.client.facet import SCHEMA_URI, BaseFacet
+
+from astro.constants import FileType
+from astro.table import Column, Metadata
+
+
+@attr.s
+class InputDatasetFacet(BaseFacet):
```

Comment:
I think we should rename this to also include File in the name somewhere since this input facet is specific to files. There could be other input facets too like for table.

Reply:
[Done](https://github.com/astronomer/astro-sdk/pull/1025/commits/ded9b02530ba670b490ef371eac36cf693133118)